c7n/resources/vpc.py

```diff
@@ -85,6 +85,39 @@ def process(self, resources):
                 client.modify_vpc_attribute(**params)
 
 
+@Vpc.action_registry.register('delete')
+class DeleteVpc(BaseAction):
+    """Delete a VPC
+
+    For example, if you want to delete an empty VPC
+
+    :example:
+
+      .. code-block:: yaml
+
+        - name: aws-ec2-vpc-delete
+          resource: vpc
+          action:
+            - type: delete
+
+    """
+    schema = type_schema('delete')
+    permissions = ('ec2:DeleteVpc')
```

Comment:
this needs a trailing comma after the string.

Reply:
will add that. There is an incoming change to fix the vpcid.

Reply:
thanks to @pgedara who sorted this out.